### ISSUE TITLE
Uncaught SyntaxError: Unexpected token u

### ISSUE THREAD

**Issue by pjayala** [mcanders/atom-hidpi#4]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 0.210.0
**System**: Ubuntu 15.04
**Thrown From**: [hidpi](https://github.com/mcanders/atom-hidpi) package, v0.6.0
### Stack Trace

Uncaught SyntaxError: Unexpected token u

```
At file:///opt/atom/resources/app.asar/static/index.html#%7B%22locationsToOpen%22%3A%5B%7B%22pathToOpen%22%3A%22%2Fhome%2Fpeayala%2Fscansafe%2Fauthentication-service%22%7D%5D%2C%22bootstrapScript%22%3A%22%2Fopt%2Fatom%2Fresources%2Fapp.asar%2Fsrc%2Fwindow-bootstrap.js%22%2C%22resourcePath%22%3A%22%2Fopt%2Fatom%2Fresources%2Fapp.asar%22%2C%22devMode%22%3Afalse%2C%22safeMode%22%3Afalse%2C%22includeDeprecatedAPIs%22%3Afalse%2C%22appVersion%22%3A%220.210.0%22%2C%22initialPaths%22%3A%5B%22%2Fhome%2Fpeayala%2Fscansafe%2Fauthentication-service%22%5D%7D:1

SyntaxError: Unexpected token u
    at Object.parse (native)
    at Hidpi.update (/home/peayala/.atom/packages/hidpi/lib/hidpi.coffee:85:32)
```
### Commands

```
     -0:18 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-atom-dark-syntax.theme-seti-ui)
```
### Config

``` json
{
  "core": {
    "themes": [
      "seti-ui",
      "atom-dark-syntax"
    ],
    "disabledPackages": [
      "hidpi"
    ],
    "autoHideMenuBar": true
  },
  "hidpi": {}
}
```
### Installed Packages

``` coffee
# User
angularjs, v0.3.0
git-tab-status, v1.9.2
jshint, v1.3.7
language-diff, v0.3.0
open-recent, v2.2.4
seti-ui, v0.7.1
travis-ci-status, v0.15.1

# Dev
No dev packages
```

**Comment by cart** [comment]
I am not sure what to make of this error.  According to your config, the atom-hidpi package is disabled.   Additionally, it says you are running atom-hidpi v0.6.0 and the error occurred on line 85 of hidpi.coffee, however in v0.6.0 hidpi.coffee only contains 79 lines.  Can you provide some context for this error?

**Comment by cart** [comment]
This seems like it should be fixed by a4521cf0f44de8684be10103a0acf3a17b09bafe.  Let me know if you are still having problems.

**Comment by pjayala** [comment]
It worked just fine this time!
